apps/server/src/modules/board/service/internal/board-node-copy.service.ts

```diff
@@ -283,35 +285,60 @@ export class BoardNodeCopyService {
 		return Promise.resolve(result);
 	}
 
-	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	async copyExternalToolElement(original: ExternalToolElement, context: CopyContext): Promise<CopyStatus> {
-		const copy = new ExternalToolElement({
+		let copy: ExternalToolElement | DeletedElement;
+		copy = new ExternalToolElement({
 			...original.getProps(),
 			...this.buildSpecificProps([]),
 		});
 
-		let status: CopyStatusEnum;
-		if (this.configService.get('FEATURE_CTL_TOOLS_COPY_ENABLED') && original.contextExternalToolId) {
-			const linkedTool = await this.contextExternalToolService.findById(original.contextExternalToolId);
+		if (!this.configService.get('FEATURE_CTL_TOOLS_COPY_ENABLED') || !original.contextExternalToolId) {
+			const copyStatus: CopyStatus = {
+				copyEntity: copy,
+				type: CopyElementType.EXTERNAL_TOOL_ELEMENT,
+				status: CopyStatusEnum.SUCCESS,
+			};
 
-			if (linkedTool) {
-				const contextExternalToolCopy: ContextExternalTool =
-					await this.contextExternalToolService.copyContextExternalTool(linkedTool, copy.id);
+			return Promise.resolve(copyStatus);
+		}
 
-				copy.contextExternalToolId = contextExternalToolCopy.id;
+		const linkedTool = await this.contextExternalToolService.findById(original.contextExternalToolId);
+		if (!linkedTool) {
+			const copyStatus: CopyStatus = {
+				copyEntity: copy,
+				type: CopyElementType.EXTERNAL_TOOL_ELEMENT,
+				status: CopyStatusEnum.FAIL,
+			};
 
-				status = CopyStatusEnum.SUCCESS;
-			} else {
-				status = CopyStatusEnum.FAIL;
-			}
+			return copyStatus;
+		}
+
+		const contextToolCopyResult: ContextExternalTool | CopyContextExternalToolRejectData =
+			await this.contextExternalToolService.copyContextExternalTool(linkedTool, copy.id, context.targetSchoolId);
+
+		let copyStatus: CopyStatusEnum = CopyStatusEnum.SUCCESS;
+		if (contextToolCopyResult instanceof CopyContextExternalToolRejectData) {
+			copy = new DeletedElement({
+				id: new ObjectId().toHexString(),
+				path: copy.path,
+				level: copy.level,
+				position: copy.position,
+				children: [],
+				createdAt: new Date(),
+				updatedAt: new Date(),
+				deletedElementType: ContentElementType.EXTERNAL_TOOL,
+				title: contextToolCopyResult.externalToolName,
+			});
+
+			copyStatus = CopyStatusEnum.FAIL;
 		} else {
-			status = CopyStatusEnum.SUCCESS;
+			copy.contextExternalToolId = contextToolCopyResult.id;
 		}
 
 		const result: CopyStatus = {
 			copyEntity: copy,
 			type: CopyElementType.EXTERNAL_TOOL_ELEMENT,
```

Comment:
The copy status type is not dependable on the copy entity, it is being used by the frontend to find out what is being attempted to copy. So in this function, this should always be `EXTERNAL_TOOL_ELEMENT` even though the copied entity is a deleted placeholder